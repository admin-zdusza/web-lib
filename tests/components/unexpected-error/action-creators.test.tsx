import {
  unexpectedErrorAcknowledged,
  unexpectedErrorOccured,
} from "../../../src/components/unexpected-error/actions-creators";

it("should create unexpected error occured", async () => {
  expect(unexpectedErrorOccured()).toMatchSnapshot();
});

it("should create unexpected error acknowledged", async () => {
  expect(unexpectedErrorAcknowledged()).toMatchSnapshot();
});
