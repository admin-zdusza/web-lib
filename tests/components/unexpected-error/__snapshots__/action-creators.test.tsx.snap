// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should create unexpected error acknowledged 1`] = `
Object {
  "type": "UNEXPECTED_ERROR_ACKNOWLEDGED",
}
`;

exports[`should create unexpected error occured 1`] = `
Object {
  "type": "UNEXPECTED_ERROR_OCCURED",
}
`;
